### PR TITLE
Add multiFieldUpdate option & Fix the try statement

### LINKED ISSUE #29: Make it possible to disable multiple dropdown update
Would it be worth considering making the feature from https://github.com/mit-cml/workspace-multiselect/issues/3 optional, instead of always on?

Defaulting to true is probably fine, but it would be nice to be able to turn it off, e.g. add a config option to guard this code:

image

---------------

**Rationale**

The idea in #3 does sound quite nice but it is not always safe to do.

Blocks with mutators may have the same `type_id` but aren't guaranteed to have the same fields. In my application, the code above crashes on `setFieldValue`.

And the approach also doesn't currently scale to dropdowns in the shape of shadow blocks, so I think I would prefer to turn this option off for consistency.

Thanks!

### LINKED ISSUE #33: Error when changing a block if it is no longer in the workspace
We have a mechanism in our project that replaces blocks when a property is changed in a drop-down list, which initiates a block change event. Since the old block is deleted, an error occurs from the library, since the program cannot find the necessary block. This occurs in the multiselect file on line 242. I suggest at the beginning of the try statement to add a check for the existence of a block on the workspace and if there is no such block, then use return.
If you approve of this behavior, I will create a PR
image

### PR DESCRIPTION
**Pull request by HollowMan6**
Resolves #29
Resolves #33 

Make it possible to disable multiple dropdown update, default is enabled
Fix the try statement to make it work

### CODE REVIEW

**Comment by ewpatton** [comment]
@laurensvalk @ bbrudastyy Can you both confirm this PR fixes your issues? I'd like to merge it and deploy a new version.

File: src/multiselect.js [hunk 1]
```diff
@@ -251,6 +251,9 @@ export class Multiselect {
             block.setFieldValue(e.newValue, e.name);
           }
         });
+      } catch {
+        // Avoid errors when changing a block if it is no longer in the workspace.
+        // https://github.com/mit-cml/workspace-multiselect/issues/33
```
Comment (ewpatton) [inline src/multiselect.js:256]: We may want to consider using `console.warn()` here to let the developer know this situation has occurred.

Comment (HollowMan6) [reply src/multiselect.js:256]: Updated

**Comment by HollowMan6** [comment]
> I do get these warnings. Maybe it could be addressed as part of #34 or separately?
> 
> image

Fixed this in #34

**Comment by laurensvalk** [comment]
Thank you. If I take this PR (#30) and `cherry-pick` #34 on top, the warnings are gone.

**Pull request merged**